### DISCUSSION TITLE
Pull request

### DISCUSSION THREAD

**Discussion by DubaiJewelersNFT** [community/community#31637] [Other Feature Feedback, Questions, & Ideas]
i have tried to create a new pull request, each time i get a mesage that This repository has been archived by the owner. It is now read-only. am really tired , any help please please

**Comment by airtower-luna** [comment]
The repository being archived means the owner doesn't work on it any more. It's just around so you can see what used to be if you want. So you can't make a PR there.

**Reply by DubaiJewelersNFT** [reply]
is their any way i can go round it, just trying to create a social token for my lil boy, i even create another one still got the samme message

**Reply by DubaiJewelersNFT** [reply]
how can use create one that is editable. thanks

**Comment by riversdiniz** [comment]
When creating the project did you create a branch? if yes give the command git branch --list to see which branch you are on, this usually happens when you are creating a branch and you accidentally went back to main.
